### PR TITLE
TST use MNE sample data

### PR DESCRIPTION
**Pull request by jasmainak**
@EtienneCmb Since you made PR from `master` I can't push directly (and maybe I don't have the right permissions either ...). This should get us started. But no caching is there yet

### CODE REVIEW

**Review by EtienneCmb — COMMENTED** [review]
Nice. Now, do you know how to make it works on Travis?

**Comment by jasmainak** [comment]
Travis is not complaining about the data anymore ... it's something else

**Comment by EtienneCmb** [comment]
pyqt5 should be installed because it's used as a matplotlib backend

**Comment by EtienneCmb** [comment]
@jasmainak test pass :)

File: .travis.yml [hunk 1]
```diff
@@ -19,6 +19,8 @@ matrix:
 
 
 before_install:
+    # See https://docs.travis-ci.com/user/gui-and-headless-browsers/#Using-xvfb-to-Run-Tests-That-Require-a-GUI :
+    - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16"
```
Comment (jasmainak) [inline .travis.yml]: is this for `visbrain`?

For matplotlib there is a simpler solution ...

Comment (EtienneCmb) [reply .travis.yml]: Well, not only for visbrain. See for example for [mne](https://github.com/mne-tools/mne-python/blob/master/.travis.yml#L52)

Comment (jasmainak) [reply .travis.yml]: yes that's for `mayavi` but I would suggest adding it when needed. It shouldn't be necessary for PSD plots

Comment (EtienneCmb) [reply .travis.yml]: Indeed, tests are still working without this line.

Comment (EtienneCmb) [reply .travis.yml]: Now the only thing that need to be add to travis is : 
```
cache:
  directories:
  - $HOME/mne_data/
```
?

File: .travis.yml [hunk 2]
```diff
@@ -44,6 +46,12 @@ install:
     - cd ${SRC_DIR}
     - pip install -e .
 
+before_script:
+    # See https://docs.travis-ci.com/user/gui-and-headless-browsers/#Using-xvfb-to-Run-Tests-That-Require-a-GUI :
+    - "export DISPLAY=:99.0"
+    - "sh -e /etc/init.d/xvfb start"
+    - sleep 3 # give xvfb some time to start
```
Comment (jasmainak) [inline .travis.yml]: so this is also not needed ...

Comment (EtienneCmb) [reply .travis.yml]: You can try to remove it but you had an issue related to xvfb with your first PR if I remember correctly

**Comment by jasmainak** [comment]
@EtienneCmb it works now without `xvfb`. Just need to use a different `matplotlib` backend.

**Comment by jasmainak** [comment]
Merge if you are happy

**Pull request merged**